app/src/main/res/values/strings.xml

```diff
@@ -1463,6 +1463,7 @@ Alternatively, you can leave a note (with a photo)."</string>
     <string name="quest_surface_detailed_answer_impossible_confirmation">Are you sure that it is impossible to specify the surface? Note the “Differs along the way” answer option that allows you to cut the way where the surface changes. Please use “Can’t say” if there is a single surface but it is not available as an answer.</string>
     <string name="quest_surface_detailed_answer_impossible_description">Please briefly specify the nature of the surface here, for example “sandy with patches of cobblestone”. The text length is limited to 255 characters.</string>
     <string name="quest_surface_detailed_answer_impossible_title">Describe surface</string>
+    <string name="quest_surface_tractypeMismatchInput_confirmation_description">This answer is not matching what is already mapped about this road. Are you sure that you are answering about the correct road? Are you sure that you personally surveyed along entire length, as always needed?</string>
```

Comment:
A bit long and sounds a bit untrusting/condescending towards the user: "personally ... as always needed".

Thinking about this, I am not sure if a confirmation is necessary or useful at all. I mean, there is also no confirmation on a resurvey when the user selected a different surface than what was selected before. After all, the user is on-site, whatever he sees is the ground truth now, regardless of what was tagged before.

Reply:
See #4063 - there are at least some cases where this resulted in wrong data.

And actually, if there is significant change in `surface` (`asphalt` <-> `dirt`) I would also consider asking. (do not ask on transitions like `dirt` <-> `grass` or `gravel` <-> `compacted`)

Reply:
Hm, I've read through #4063 and I do not find it so convincing.

First of all, it is not really clear why for the case which @mcliquid mentioned the wrong surface was selected by a user. We don't know if it is vandalism, fat fingers, carelessness etc.

If there was not any malintent however, then wouldn't this message need to be displayed **always** to reduce the number of wrong answers? Because whether or not something else was recorded before or nothing was recorded before, the situation is the same. The user is on-site now and can answer about the current situation.

Reply:
There is difference between claiming that surface was `asphalt` and is still `asphalt` (very likely), claiming that surface changed from `dirt` to `asphalt` (rare) and claiming that surface changed from `asphalt` to `dirt` (really rare and unusual).

It may make sense to ask for confirmation in cases which are rare or very rare, while not asking for typical ones.

For cases where no surface data was tagged we have no indicator to suspect that it is case of a a rare case worth checking.

Also, in cases where no surface was tagged user is at worst adding false data which is less bad that destroying correctly mapped data.

Reply:
Hmm, okay